step-strikers-app/step-strikers-app/CodeEntryViewController.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  CodeEntryViewController.swift
+//  step-strikers-app
+//
+//  Created by Alekhya Kuchimanchi on 3/1/23.
+//
+
+import UIKit
+
+class CodeEntryViewController: UIViewController {
+    
+    var popUp:UIView?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+        assignBackground()
+        
+        // background faded shoe
+        createImage(x: 0, y: 250, w: 375, h: 375, name: "Faded Emblem")
+        
+        // giant enter code text
+        createLabel(x: 50, y: 200, w: 300, h: 200, font: "iso8", size: 45, text: "ENTER CODE:", align: .center)
+        
+        // textfield for people to enter code
+        // TODO: save this to check later
+        let textField = UITextField(frame: CGRect(x: 100, y: 350, width: 200, height: 40))
+        textField.backgroundColor = UIColor.white
+        textField.layer.borderColor = UIColor.brown.cgColor
+        textField.layer.borderWidth = 2.0
+        self.view.addSubview(textField)
+        
+        // add settings button to bottom right corner
+        createSettingsButton(x: 325, y: 775, width: 40, height: 40)
+        // create the back button to go to battle meny again
+        let backButton = UIButton()
+        backButton.frame = CGRect(x: 160, y:700, width:75, height:60)
+        backButton.setTitle("BACK", for:UIControl.State.normal)
+        backButton.titleLabel!.font = UIFont(name: "munro", size: 20)
+        backButton.setTitleColor(.black, for:.normal)
```

Comment:
Left in by mistake? The title color is set to brown immediately afterward.

```suggestion
        backButton.setTitleColor(.black, for:.normal)
```